src/content/overview/introduction.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Introduction to Giant Swarm
+description: Giant Swarm give platform engineering teams the ability to build cloud-native developer platforms and operate them without much hassle.
+weight: 10
+menu:
+  main:
+    parent: overview
+user_questions:
+  - What is a cloud-native developer platform?
+  - How does Giant Swarm help engineering teams?
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-horizon
+last_review_date: 2024-03-12
+---
+
+Giant Swarm's mission is to empower our customers to build their own cloud-native [developer platform](https://tag-app-delivery.cncf.io/whitepapers/platforms/#what-is-a-platform). As engineering teams grow and their environments expand, they often face exponentially growing complexity within the cloud-native ecosystem. This can be overwhelming. We believe that in most cases, having a [unified platform](https://tag-app-delivery.cncf.io/whitepapers/platforms/#why-platforms) plays a crucial role in keeping or gaining control over the entropic nature of such complex environments, while also significantly enhancing the users experience in using them.
+
+Ultimately, developers want to deploy an app into production without having to worry too much about security or compliance, but still have good monitoring and deployment options. Our goal is to help the engineers who build and maintain the platform meet developers' expectations without compromising their organizations' security posture.
```

Comment:
- At best reword without plural apostrophe (`developers'`, `organizations'`)
- We're missing a statement or diagram to show _platform team_ vs. _developers_ (and a typical ratio i.e. much fewer platform engineers than devs)